semantic-conventions/src/opentelemetry/semconv/model/utils.py

```diff
@@ -13,7 +13,7 @@
 #   limitations under the License.
 
 import re
-from typing import Tuple
+from typing import Tuple  # noqa: F401
```

Comment:
used implicitly, but flake does not understand it

Reply:
I think the "proper" solution would be to switch from legacy `#type:` comments to using actual annotations.

Reply:
I created https://github.com/open-telemetry/build-tools/issues/234 as a follow-up but will merge this PR as-is for now.
cc @lmolkova